pkg/volume/csi/csi_client_test.go

```diff
@@ -1,173 +0,0 @@
-/*
```

Comment:
Would be a good idea to have some tests for the newly added client code.

Reply:
Will add some tests.